surfer/viz.py

```diff
@@ -1737,6 +1748,45 @@ def add_text(self, x, y, text, name, color=None, opacity=1.0,
         if justification is not None:
             text.property.justification = justification
 
+    def add_text3d(self, x, y, z, text, name, color=None, opacity=1.0,
+                 row=-1, col=-1, font_size=None, justification=None):
```

Comment:
Do we use this `row`, `col` parameterization anywhere else in pysurfer? Ideally if you're going to add a new method it should match the existing API as closely as possible.